Makefile

```
@@ -160,7 +161,12 @@ tetragon-bpf-container:
 verify: tetragon-bpf
 	sudo contrib/verify/verify.sh bpf/objs
 
-.PHONY: tetragon tetra tetragon-operator tetragon-bench
+.PHONY: generate-flags tetragon tetra tetragon-operator tetragon-bench
+generate-flags: tetragon
+	echo "\`\`\`text" > docs/content/en/docs/reference/daemon-flags.md
```

```suggestion
	echo "\`\`\`" > docs/content/en/docs/reference/daemon-flags.md
```
Only nit would be to remove the `text` for it to render like output.

---

Alright removed ;-)